src/main/java/ch/wisv/areafiftylan/model/Ticket.java

```diff
@@ -63,11 +64,27 @@ public Ticket() {
         //JPA Only
     }
 
+    public void setUpForTransfer(User u){
+        this.setLockedForTransfer(true);
```

Comment:
You're locking the ticket for transfer if you want to set up a ticket for transfer?